src/WorldMap.cpp

```diff
@@ -165,10 +136,7 @@ WorldMap::TileLayer::TileLayer(const std::vector<std::vector<int>>& tileMapData,
 		for (int col = 0; col < mapSize.x; ++col)
 		{
 			const int tileDrawID = tileMapData[row][col]; //Get ID for tile
-			if (tileDrawID > 0)
-			{
-				m_tileMap.emplace_back(sf::Vector2f(col, row), tileSheet, tileDrawID);
-			}
+			m_tileMap.emplace_back(sf::Vector2f(col, row), tileSheet, tileDrawID);
```

Comment:
I thought this check was still necessary.  If the ID is 0, the tile location on the tile sheet won't exist.  In our code it looks like that will set the tile sprite's texture rectangle outside the texture - not sure how SFML will handle that.